src/utils/Options/index.js

```diff
@@ -2,10 +2,12 @@ import services from "./services";
 import domain from "./domain";
 import regions from "./regions";
 import other from "./other";
+import urban from "./urban";
 
 export default {
   other,
   regions,
   services,
-  domain
+  domain,
+  urban,
```

Comment:
noticed extra spaces in the beginning of the string in urban, better to trim it off